media-proxy/include/mesh/st2110.h

```diff
@@ -0,0 +1,57 @@
+#ifndef ST2110_H
+#define ST2110_H
+
+#include <thread>
+#include <bsd/string.h>
+#include <arpa/inet.h>
+#include <mtl/st_pipeline_api.h>
+#include <mtl/st30_pipeline_api.h>
+
+#include "conn.h"
+#include "mesh_dp.h"
+#include "logger.h"
+
+namespace mesh::connection {
+
+#define ST_APP_PAYLOAD_TYPE_ST30 (111)
+#define ST_APP_PAYLOAD_TYPE_ST20 (112)
+#define ST_APP_PAYLOAD_TYPE_ST22 (114)
+
+/**
+ * ST2110
+ *
+ * Base abstract class of ST2110. ST2110Rx/ST2110Tx
```

Comment:
```suggestion
 * Base abstract class of SPMTE ST2110-xx bridge. ST2110Rx/ST2110Tx
```

Reply:
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/261/commits/2479ca6f6d060db56009026a4c727be98472c6dc